setup/freebsd/ansible-vars.yaml

```diff
@@ -4,8 +4,9 @@ init_script_path: /usr/local/etc/rc.d/jenkins
 ssh_users:
   - rvagg
   - jbergstroem
+  - joaocgreis
 packages:
-  - openjdk-jre
+  - openjdk
```

Comment:
Mostly for my own curiosity, is this just because the JDK package has saner defaults? Or is the compiler actually required?